### PR TITLE
[chore][log] Opt log, revert some log introduced by #25739

### PR DESCRIPTION
**Pull request by zhiqiang-hhhh**
Opt log, revert some log introduced by https://github.com/apache/doris/pull/25739

### CODE REVIEW

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

File: be/src/runtime/fragment_mgr.cpp [hunk 1]
```diff
@@ -453,12 +452,13 @@ void FragmentMgr::_exec_actual(std::shared_ptr<PlanFragmentExecutor> fragment_ex
         std::lock_guard<std::mutex> lock(_lock);
         _fragment_instance_map.erase(fragment_executor->fragment_instance_id());
 
-        LOG_INFO("Instance {} finished",
```
Comment (yiguolei) [inline be/src/runtime/fragment_mgr.cpp:456]: not remove these logs, it is useful

Comment (zhiqiang-hhhh) [reply be/src/runtime/fragment_mgr.cpp:456]: Fixed, add back

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

**Review by HappenLee — DISMISSED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**